lcu_driver/utils.py

```diff
@@ -13,9 +13,18 @@ def parse_cmdline_args(cmdline_args) -> Dict[str, str]:
 
 
 def _return_ux_process() -> Generator[Process, None, None]:
-    for process in process_iter():
-        if process.status() == STATUS_ZOMBIE:
-            continue
+    for process in process_iter(attrs=["cmdline"]):
+        try:
+            if process.status() == STATUS_ZOMBIE:
+                continue
 
-        if process.name() in ['LeagueClientUx.exe', 'LeagueClientUx']:
-            yield process
+            cmdline: List[str] = process.info.get("cmdline", [])
+
+            if process.name() in ["LeagueClientUx.exe", "LeagueClientUx"]:
+                yield process
+
+            if cmdline and cmdline[0].endswith("LeagueClientUx.exe"):
```

Comment:
Are you able to add a small comment before the `if` statement to remember why that condition is needed for my own future reference?

Reply:
Sure i can do that